internal/repository/article/article.go

```diff
@@ -0,0 +1,85 @@
+package article
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"news/internal/entity"
+	"news/internal/repository/mgdb"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+type ArticleRep struct {
+	articleTitle *mongo.Collection
+	article      *mongo.Collection
+}
+
+func NewArticleRep(client *mgdb.MongoDriver) *ArticleRep {
+	return &ArticleRep{
+		articleTitle: client.CreateCollection("news", "articleTitle"),
```

Comment:
does it really create a collection? if yes then this is something done in migrations, no?

Reply:
If not exist - create as i see. But you are right bad name for method, i changed to `GetCollection`

Reply:
fixed: add migration